### PR TITLE
Increase kubernetes memory.

### PR DESCRIPTION
**Pull request by schmmd**

### CODE REVIEW

File: kubernetes-webdemo.yaml [hunk 1]
```diff
@@ -23,13 +23,13 @@ spec:
               # One-half of a CPU.
               cpu: 500m
               # 500 megabytes.
-              memory: 1000Mi
+              memory: 2000Mi
```
Comment (joelgrus) [inline kubernetes-webdemo.yaml]: fix comment

File: kubernetes-webdemo.yaml [hunk 2]
```diff
@@ -23,13 +23,13 @@ spec:
...
             # "requests" specify how much your container will be granted as a baseline.
             requests:
               # One-fifth of a CPU.
               cpu: 200m
               # 250 megabytes.
-              memory: 1000Mi
+              memory: 2000Mi
```
Comment (joelgrus) [inline kubernetes-webdemo.yaml]: fix comment

**Comment by schmmd** [comment]
Actually--I'm going to bundle some CPU upgrades with this PR.

**Comment by schmmd** [comment]
@joelgrus the comments are doomed to be inaccurate, so I deleted them.

The demo died when I upped the workers to 4, so I backed it off to 1 again.

**Comment by joelgrus** [comment]
that's not good, I'll investigate tomorrow

**Pull request merged**